core/dbt/context/base.py

```diff
@@ -114,11 +114,10 @@ def get_datetime_module_context():
     }
 
 
-def get_context_modules():
-    return {
-        'pytz': get_pytz_module_context(),
-        'datetime': get_datetime_module_context(),
-    }
+CONTEXT_MODULES: Dict[str, Dict[str, Any]] = {
```

Comment:
Can you keep this as a global function that returns a dict, instead of a global dict? I have a good reason and a bad/lazy reason:
 - I don't want us to share the same dict into every jinja environment. I think jinja does a shallow copy of the context and I want to build a fresh one every time we generate a context, since dicts are mutable.
 - I want to minimize merge conflicts with my other branch that makes the same change :)

Reply:
Sure, I've reverted the change.